flexget/api/core/server.py

```diff
@@ -164,7 +164,7 @@ def get(self, session=None):
 
 @server_api.route('/config/')
 class ServerConfigAPI(APIResource):
-    @etag
+    @etag(cache_age=0)
```

Comment:
This is the default value, so there's no need to explicitly set it here. It could be argued that `cache_age` shouldn't have a default at all though.